controllers/quiz.controller.js

```diff
@@ -0,0 +1,32 @@
+const { Quiz } = require("../models/quiz.model");
+
+const loadQuizBank = async (req, res) => {
```

Comment:
Instead of `loadQuizBank` name it something different, like `addNewQuiz`, it would be much clear :)

Reply:
done